lib/ExecutionEngine/ExecutionEngine.cpp

```diff
@@ -103,7 +103,13 @@ void glow::updateInputPlaceholdersByName(PlaceholderBindings &bindings,
          "The number of inputs does not match the number of Placeholders");
 
   for (int i = 0, e = ph.size(); i < e; i++) {
-    Placeholder *p = mod->getPlaceholderByName(ph[i]);
+    // Legalize the name.
+    std::string legalName;
```

Comment:
there are several places we do name legalization (see Graph.cpp), I'd suggest to place this in a Support lib and reuse in all those places.